hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,164 @@
+defmodule Hangman.Game do
+    @moduledoc """
+    Hangman game implementation.
+    """
+
+    ### Game state struct ###
+
+    alias Hangman.Game
+    defstruct(
+        # DEFAULTS
+        status:     :initializing,
+        guessed:    %MapSet{},
+        turns_left: 7,
+
+        # PLACEHOLDERS (values will be determined during game)
+        word:       "N/A", 
+        last_guess: "N/A"
+    )
+
+    ### Public Functions (Called from API) ###
+
+    @spec new_game(binary()) :: struct()
```

Comment:
So, you are the only person so far to create a `new_game/1` function to allow you to do testing. 

I am so delighted that I'm giving you a +6.

Then you go and copy-and-paste the code for the two functions, rather than doing

~~~ elixir
    def new_game() do
        new_game(Dictionary.random_word())
    end

    def new_game(word) do 
        %Game{ 
            word: word |> String.downcase() 
        }
    end
~~~

So I'm clawing back 2 of those 6 points... :)